admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -610,6 +610,13 @@ var specificPrices = (function() {
     });
   }
 
+  function resetForm() {
+    /**
```

Comment:
https://github.com/airbnb/javascript#comments

Multiline comments will only be used for methods & functions descriptions. Not for this.